### ISSUE TITLE
Develop Branch compatibility

### ISSUE THREAD

**Issue by Hedg0305** [boazpoolman/strapi-plugin-sitemap#46]
Hi, I'd like to leave you a question, is the development branch compatible with the Strapi v3.6.X? 
I'm asking it because as I installed it, there were some dependencies that my project didn't have such as the @strapi/helper-plugin, and after installing these dependencies and building the project again, it didn't open, so, is this branch only available for versions >v4?

**Comment by boazpoolman** [comment]
Hey @Hedg0305,
You guessed it right; the develop branch is the migrated plugin for Strapi v4. For Strapi v3 please refer to the master branch, or the latest npm release (1.2.5).

**Comment by Hedg0305** [comment]
Hey @boazpoolman, yesterday I was running some tests in the Strapi v4 and installed your plugin to test the sitemap with localizations, but I o don't know if it has what I was looking for.
This is the scenario:
The site that I'm working on has multiple domains, but I didn't find a way to configure it in your plugin, so I'd like to ask you if it's possible to configure it in the current version of your development branch or it's something not implemented yet.

**Comment by boazpoolman** [comment]
Hey @Hedg0305,
So if I understand correctly you want to generate multiple sitemaps, each with a different hostname?
This is a feature that is not implemented yet. You can only generate 1 sitemap as of now.

I have been thinking about allowing to setup multiple sitemaps for multiple domains. But as multi domain is not natively supported in Strapi, I have yet to figure out a good way to implement this.

If you don't mind me asking; in your Strapi project, how do you specify the domain to publish a page to? Do you have a domain relation on your page collections?

**Comment by Hedg0305** [comment]
I query the content according to the domain with the help of NextJS i18n locales.
Next allows you to configure a default language to each domain and then with this language I can make a query at Strapi to get the correct content.

**Comment by boazpoolman** [comment]
Okay, so if I understand correctly;
This is one website, but each locale is hosted on a different domain?

Like
```
en.domain.com // english website
fr.domain.com // french website
de.domain.com // german website
```

**Comment by Hedg0305** [comment]
Yep, that's it.

**Comment by boazpoolman** [comment]
Allright. Let me do some research on this.
Easiest solution would be to allow you to set a different hostname for each language (which is not yet possible in th plugin).
But as far as I'm concerned multiple domains in one sitemap.xml is a no-go according to Google.

I'll get back to you on this.

**Comment by boazpoolman** [comment]
In the meantime; is there a specific reason you would not host your websites with the locales on a subpath, instead of subdomain?

Like:
```
domain.com/en // english website
domain.com/fr // french website
domain.com/de // german website
```

**Comment by Hedg0305** [comment]
The website is already made, but they used to use an old CMS called EZ + PHP, and I'm working on the migration to Strapi + NextJS and their request was to keep the old URL system.

**Comment by boazpoolman** [comment]
Makes sense.
Again; I'll do some research and get back to you.

**Comment by Hedg0305** [comment]
You're welcome.
And your plugin is working nicely, good job. 😁

**Comment by boazpoolman** [comment]
Seems like it is possible to add multiple domains/subdomains to one sitemap.xml.
Source: https://developers.google.com/search/docs/advanced/sitemaps/large-sitemaps?hl=en&visit_id=637733707241955803-3025162350&rd=1#manage-sitemaps-for-multiple-sites

You might need to make sure you verify ownership for each domain in Google Seach Console.
But other than that this might work!

**Comment by boazpoolman** [comment]
@Hedg0305 I added this feature to the develop branch.
Could you test it out in your local environment?

**Comment by boazpoolman** [comment]
Released in [v2.0.0](https://github.com/boazpoolman/strapi-plugin-sitemap/releases/tag/2.0.0) for the Strapi v4 version of `strapi-plugin-sitemap`.
